internal/nginx/config/servers.go

```diff
@@ -45,13 +45,15 @@ func createSSLServer(virtualServer dataplane.VirtualServer) http.Server {
 		return createDefaultSSLServer()
 	}
 
+	locs := createLocations(virtualServer.PathRules, 443)
+
 	return http.Server{
 		ServerName: virtualServer.Hostname,
 		SSL: &http.SSL{
 			Certificate:    virtualServer.SSL.CertificatePath,
 			CertificateKey: virtualServer.SSL.CertificatePath,
 		},
-		Locations: createLocations(virtualServer.PathRules, 443),
```

Comment:
Could probably revert this change now that these are separated.

Reply:
Thanks, missed this!! 😄